CHANGELOG.md

```diff
@@ -1,16 +1,22 @@
 Changelog
 =========
 
+unreleased
+----------
+
+* **2016-05-25** Use "auto" for publish workflow enabled flag. If auto and
+  CmfCoreBundle is not instantiated, the PWF is not loaded.
+
 2.0.0
 -----
 
 2.0.0-RC1
 ---------
 
-* **2014-11-03**: [BC BREAK] The CreateMenuItemFromNodeEvent class no longer has a getFactory method, inject the factory as a service instead
-* **2014-11-01**: [BC BREAK] The voters now need to use the voter mechanism of KnpMenu.
-* **2014-10-31**: [BC BREAK] The cmf_menu.factory service has been removed, knp_menu.factory should be used instead
-* **2014-10-30**: [BC BREAK] PhpcrMenuProvider now requires the NodeLoader as first argument instead of a FactoryInterface
+* **2015-11-03**: [BC BREAK] The CreateMenuItemFromNodeEvent class no longer has a getFactory method, inject the factory as a service instead
+* **2015-11-01**: [BC BREAK] The voters now need to use the voter mechanism of KnpMenu.
+* **2015-10-31**: [BC BREAK] The cmf_menu.factory service has been removed, knp_menu.factory should be used instead
+* **2015-10-30**: [BC BREAK] PhpcrMenuProvider now requires the NodeLoader as first argument instead of a FactoryInterface
```

Comment:
these 4 items were actually added in 2014 (we've not done a CMF release for more than a year).